api/tests/test_add_response_headers.py

```diff
@@ -11,6 +11,8 @@
     'X-XSS-Protection': '1; mode=block'
 }
 
+CSP_HEADER = {'Content-Security-Policy': "default-src 'self'; style-src 'self' 'unsafe-inline'; font-src data:; img-src 'self' data:; child-src blob:; object-src 'none'; frame-ancestors 'none'; base-uri 'none';"}
+
```

Comment:
nitpick(non-blocking): every time a dev formats his/her code correctly, a puppy smiles

Reply:
Will indent properly in next revision!